contrib/README.contrib

```diff
@@ -67,6 +67,9 @@ puff/       by Mark Adler <madler@alumni.caltech.edu>
         Small, low memory usage inflate.  Also serves to provide an
         unambiguous description of the deflate format.
 
+tests/      by Chris Blume <cblume@google.com>
```

Comment:
I'd recommend "fuzz" instead of "tests"

Reply:
The complete directory structure is contrib/tests/fuzzers/*

Is that sufficient?
(In the README.contrib you only list a contact for that top level.)

Reply:
SGTM! Thanks